src/components/MenuList/MenuList.module.css

```
@@ -0,0 +1,23 @@
+.menuCategory {
+  border-bottom: 20px solid #ececec;
+  padding-bottom: 2px;
+  padding: 0 16px;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-bottom



(lint/suspicious/noShorthandPropertyOverrides)